nexus/pkg/server/sender.go

```diff
@@ -362,11 +364,11 @@ func (s *Sender) checkAndUpdateResumeState(run *service.RunRecord) error {
 	}
 
 	if s.resumeState.FileStreamOffset == nil {
-		s.resumeState.FileStreamOffset = make(map[chunkFile]int)
+		s.resumeState.FileStreamOffset = make(map[fs.ChunkFile]int)
```

Comment:
nit: but you already have a type for this map... should we just use it?

Reply:
yes. when i did that search and replace i was planning on coming back to that.

Reply:
fixed!